sdk/tests/deqp/functional/gles3/es3fSyncTests.js

```diff
@@ -33,6 +33,7 @@ goog.scope(function() {
     var deString = framework.delibs.debase.deString;
 
     /** @const {number} */ es3fSyncTests.NUM_CASE_ITERATIONS = 5;
+    /** @const {number} */ es3fSyncTests.MAX_NUM_VERIRY = 10;
```

Comment:
In es3fTransformFeedbackTests.js, the strategy used was to spin for a maximum of a certain number of seconds (5). Could that be done here too? I think it would make the test less potentially flaky; there's no reason it should take 5 seconds for a sync object to be signaled.